compiler/vertex-desc.json

```diff
@@ -440,6 +440,10 @@
         "type": "bool",
         "default": "false"
       },
+      "type_as_part_of_signature": {
+        "type": "const TypeHint *",
+        "default": "nullptr"
+      },
```

Comment:
I really don't like this field added. 
@quasilyte, how do you think: after #171 is merged, this addition becomes useless?